infra/scripts/predeploy/set-app-configuration.ps1

```diff
@@ -132,7 +114,7 @@ function Get-WorkloadKeyVault {
     $hubGroup = Get-AzResourceGroup -Name $group.Tags["HubGroupName"]
 
     # the group contains tags that explain what the default name of the kv should be
-    $keyVaultName = "kv-$($group.Tags["ResourceToken"])"
+    $keyVaultName = "kv-$($hubGroup.Tags["ResourceToken"])"
```

Comment:
Also need to update the error message a few lines later to show the correct kv name that doesn't exist.